xml-parser/src/main/java/org/uniprot/core/xml/dbreference/UniParcCrossReferenceConverter.java

```diff
@@ -85,6 +85,7 @@ public DbReference toXml(UniParcCrossReference uniObj) {
         xmlObj.getProperty().addAll(properties);
         if (Utils.notNullNotEmpty(uniObj.getProperties())) {
             uniObj.getProperties().stream()
+                    .filter(prop -> !SOURCES.equals(prop.getKey()))
                     .map(prop -> createProperty(prop.getKey(), prop.getValue()))
                     .forEach(val -> xmlObj.getProperty().add(val));
         }
```

Comment:
Make sure it doesn't impact our data load. I mean, we should not filter sources during load.